app/views/monitoring.html

```diff
@@ -260,6 +262,108 @@
+                    </div>
+                    <div class="list-group-item list-group-item-expandable"
+                         ng-repeat-start="set in filteredStatefulSets track by (set | uid)"
+                         ng-click="toggleItem($event, this, set)"
+                         ng-class="{'expanded': expanded.statefulSets[set.metadata.name]}">
+                      <div class="list-view-pf-checkbox">
+                        <button class="sr-only">{{expanded.statefulSets[set.metadata.name] ? 'Collapse' : 'Expand'}}</button>
+                        <span ng-if="expanded.statefulSets[set.metadata.name]">
+                          <span class="fa fa-angle-down"></span>
+                        </span>
+                        <span ng-if="!expanded.statefulSets[set.metadata.name]">
+                          <span class="fa fa-angle-right"></span>
+                        </span>
+                      </div>
+
+                      <div class="list-view-pf-main-info">
+                        <div class="list-view-pf-body">
+                          <div class="list-view-pf-description">
+                            <div class="list-group-item-heading">
+                              <a ng-href="{{set | navigateResourceURL}}"><span ng-bind-html="set.metadata.name | highlightKeywords : filterKeywords"></span></a>
+                              <small>created <span am-time-ago="set.metadata.creationTimestamp"></span></small>
+                            </div>
+                            <div class="list-group-item-text">
+                              <!-- TODO: is this the right status? -->
+                              <status-icon status="set | deploymentStatus" disable-animation></status-icon>
+                              {{set | deploymentStatus | sentenceCase}}
+                              <span ng-if="set.status.replicas !== set.spec.replicas">{{set.status.replicas}}/</span>{{set.spec.replicas}} replica<span ng-if="set.spec.replicas != 1">s</span>
+                            </div>
+                          </div>
+                          <div class="list-view-pf-additional-info">
+                            <div class="list-view-pf-additional-info-item">
+                              <span class="pficon fa-fw pficon-image"></span>
+                              <!-- TODO: not sure what to do with this, gotta figure it out -->
+                              <image-names pod-template="set.spec.template" pods="podsByOwnerUID[set.metadata.uid]"></image-names>
+                            </div>
+                          </div>
+                        </div>
+                      </div>
+                    </div>
+                    <div ng-repeat-end ng-if="expanded.statefulSets[set.metadata.name]" class="list-group-expanded-section" ng-class="{'expanded': expanded.statefulSets[set.metadata.name]}">
+
+                      <!-- TODO: add logs once we have logs subresource for statefulSets
```

Comment:
honestly i would probably just remove this completely, we dont have logs for replicasets either and it may be awhile before we do, I dont think we left any commented out code for those